installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -16,7 +16,7 @@ The following steps outline the high-level workflow on how to use the oc-mirror
 . Mirror the image set to the mirror registry by using one of the following methods:
 ** Mirror an image set directly to the mirror registry.
 ** Mirror an image set to disk, transfer the image set to the target environment, then upload the image set to the target mirror registry.
-. Install the `ImageContentSourcePolicy` and `CatalogSource` resources that were generated by oc-mirror into the cluster.
+. Configure your cluster to use resources that oc-mirror generated.
```

Comment:
```suggestion
. Configure your cluster to use the resources that oc-mirror generated.
```
Not entirely sure. Should references to oc- mirror actually be oc-mirror plugin?

Reply:
I see oc-mirror and oc-mirror plugin used interchangeably later down the page, but I can change this reference to include "plugin" to be on the safer side